### PR TITLE
Added new container: diamond-aligner

### PR DESCRIPTION
**Pull request by Facundo1224**
Changes:
- Added a new container: diamond-aligner.

Additional notes:
- Merged main into testing after pushing this change because I forgot to pull before adding the container.

### CODE REVIEW

**Comment by sbassi** [comment]
The code seems good, didn't test it yet. You need to move it into a directory named with the version name.
Instead of diamond-aligner/Dockerfile
should be:
diamond-aligner/2.1.9-debian/Dockerfile

File: diamond-aligner/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+LABEL \
+    software="diamond" \
+    base_image="debian:trixie" \
+    container="diamond-aligner" \
+    about.summary="DIAMOND is a sequence aligner for protein and translated DNA searches,\
+    designed for high performance analysis of big sequence data" \
+    about.home="https://github.com/bbuchfink/diamond/" \
+    software.version="2.1.9-1" \
+    upstream.version="2.1.10" \
```
Comment (sbassi) [inline diamond-aligner/Dockerfile]: upstream should be 2.1.9

File: diamond-aligner/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt update && apt-get install -y \
+    diamond-aligner=2.1.9-1 \
+    && apt clean && apt purge \
+    && rm -rf /var/lib/apt/lists/* /tmp/*
+
+WORKDIR /data
+
+ENTRYPOINT ["diamond"]
```
Comment (sbassi) [inline diamond-aligner/Dockerfile]: Remove Entrypoint and CMD.

File: diamond-aligner/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    about.copyright="DIAMOND authors" \
+    about.license="GPL-3.0" \
+    about.license_file="/usr/share/doc/diamond/copyright" \
+    maintainer="Facundo Mercado <facu.mercado1224@gmail.com>" \
+    about.tags="bioinformatics, sequence alignment, protein clustering"
+
+USER root
+
+# 1 warning found LegacyKeyValueFormat: "ENV key=value" should be used instead of legacy "ENV key value" format (line 21)
+ENV DEBIAN_FRONTEND=noninteractive
```
Comment (sbassi) [inline diamond-aligner/Dockerfile]: replace this line as suggested in the warning and delete the warning comment

File: diamond-aligner/Dockerfile (hunk 1 above)
Comment (sbassi) [reply diamond-aligner/Dockerfile]: in upstream, you can use:
2.1.9.163
Since this is what the program reports.

**Review by sbassi — CHANGES_REQUESTED** [review]
The file is good, it works as is, but is better without the entry point, and there are minors changes required, so please address all comments.

**Comment by Facundo1224** [comment]
Done! Changes in new commit:

- Adjusted file path to "diamond-aligner/2.1.9-debian/Dockerfile" (previously "diamond-aligner/Dockerfile")
- Corrected the "upstream.version" label to 2.1.9.163
- Removed the "ENTRYPOINT" and "CMD" layers from the Dockerfile.
- Removed leftover warning comment

**Review by sbassi — APPROVED** [review]

**Pull request merged**